### PR TITLE
`repo-header-info` - Improve alignment 

### PR DESCRIPTION
**Pull request by 134130**
## Describe
- Closes: #6967 

## Test URLs
- https://github.com/refined-github/refined-github

### Original 😱 
original

### `vertical-align: text-top` ✅ 
text-top

### `vertical-align: bottom` 🤔 
vertical-align-bottom

### CODE REVIEW

File: source/features/repo-header-info.tsx [hunk 1]
```diff
@@ -49,10 +49,12 @@ async function add(repoLink: HTMLAnchorElement): Promise<void> {
 			<a
 				href={buildRepoURL('stargazers')}
 				title={`Repository starred by ${stargazerCount.toLocaleString('us')} people`}
-				className="d-flex flex-items-center flex-justify-center mr-1 gap-1 color-fg-muted"
+				className="d-inline-block"
 			>
-				<StarIcon className="ml-1" width={12} height={12}/>
-				<span className="f6">{abbreviateNumber(stargazerCount)}</span>
+				<div className="d-inline-flex flex-items-center flex-justify-center mr-1 gap-1 v-align-text-top color-fg-muted">
```
Comment (FloEdelmann) [inline source/features/repo-header-info.tsx:54]: Can't you avoid the wrapper `<div>` by applying exactly these classes to `<a>`?

Comment (134130) [reply source/features/repo-header-info.tsx]: Thank you!
I thought It was not changed when without wrapping because of my old eyes issue.

**Comment by fregante** [comment]
The alignment in your screenshot appears to be correct, but we should also match the font size exactly

Screenshot 2

**Comment by fregante** [comment]
Also I think this isn't actually affecting Safari for some reason. It looks exactly the same. The Star alignment is also bad there 😕 


Screenshot 5

**Comment by 134130** [comment]
image

Please can you check it is aligned on safari?

**Review by fregante — APPROVED** [review]
The two classes don't change anything in Safari, do they change the alignment in Chrome?

The font size alone actually fixes the alignment though:

Screenshot 2

**Comment by 134130** [comment]
These were remnants of trying with wrapper.

**Pull request merged**